tools/bin/ci_credentials.sh

```diff
@@ -24,3 +24,7 @@ echo "$HUBSPOT_INTEGRATION_TESTS_CREDS" > airbyte-integrations/connectors/source
 
 mkdir airbyte-integrations/connectors/source-google-sheets/secrets
 echo "$GSHEETS_INTEGRATION_TESTS_CREDS" > airbyte-integrations/connectors/source-google-sheets/secrets/creds.json
+
+mkdir airbyte-integrations/connectors/source-file/secrets
+echo "$BIGQUERY_INTEGRATION_TEST_CREDS" > airbyte-integrations/connectors/source-file/secrets/gcs.json
+echo "{}" > airbyte-integrations/connectors/source-file/secrets/aws.json
```

Comment:
I don't know how to properly add new credentials (especially to some AWS accounts) so i leave this empty for now... it shouldn't be needed by the integration tests from standard tests (but are required for the custom integration tests)

Reply:
@ChristopheDuong  why would this not be needed from standard tests? don't they also perform some reads? Also, do you need this secret stored in github secrets?

Reply:
The standard tests are using public data that don't need credentials but i have custom integration tests that goes deeper with multiple auth / private buckets